pkg/cmd/pr/create/create_test.go

```diff
@@ -279,9 +279,9 @@ func Test_createRun(t *testing.T) {
 				reg.Register(
 					httpmock.GraphQL(`mutation PullRequestCreate\b`),
 					httpmock.GraphQLMutation(`
-					{ "data": { "createPullRequest": { "pullRequest": {
-						"URL": "https://github.com/OWNER/REPO/pull/12"
-					} } } }`,
```

Comment:
**Are these editor made changes or individual aesthetics?**

I'm primarily wanting to make sure the tests don't have a problem but was curious.

Reply:
Yes sorry it is my config that automatic changes this... Test are passing and no problem so far, but if you prefer I will revert those changes